README.md

```diff
@@ -21,6 +21,16 @@ You must include the jersey-crypto project to use the S/MIME framework.
 
 ## Usage ##
 
+## Warning ##
+
+Be aware of using other than default Connector implementation. There is an issue handling HTTP headers in WriterInterceptor or MessageBodyWriter<T>. If you need to change header fields do not use nor ApacheConnectorProvider nor GrizzlyConnectorProvider neither JettyConnectorProvider. 
+On the other hand, in the default transport connector, there are some restrictions on the headers, that can be sent in the default configuration.
+Since the S/MIME protocol depends on the `Content-Transfer-Encoding` header, the restrictions have to be disabled:
+    
+    System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
+
+See https://jersey.java.net/nonav/documentation/latest/user-guide.html#d0e4957 for more information on these limitations.
```

Comment:
This should probably link to a specific version of the documentation since those references (d0e4957) will change between releases: https://jersey.java.net/nonav/documentation/2.22.1/user-guide.html#d0e4957